client/tracks/index.js

```diff
@@ -28,6 +28,19 @@ function recordEvent( eventName, eventProperties ) {
 	} );
 }
 
+/**
+ * Records events from buyers.
+ *
+ * @param {string}  eventName       Name of the event.
+ * @param {Object?} eventProperties Event properties.
+ */
+function recordUserEvent( eventName, eventProperties ) {
+	if ( ! window.jpTracksAJAX ) {
+		return;
+	}
+	window.jpTracksAJAX.record_ajax_event( eventName, null, eventProperties );
```

Comment:
@malithsen I'm not 100%, but when I have Jetpack active, looks like the event triggered here following the instructions from the description the recorded event is named `jetpack_platform_checkout_registered`, and when Jetpack is not installed it's named `woocommerceanalytics_platform_checkout_registered`. Can you replicate on your end?

Reply:
Hey @a-danae this is a great catch. I could not replicate it, but I noticed your test events on Tracks live view with the incorrect prefixes. I am wondering if another plugin (Jetpack or some other) that uses `jetpack-tracking` is taking precedence over what's implemented here. In which case it will use the default `jetpack` prefix. I will try to see whether this is the case.

Reply:
Welp, I've been trying to replicate again but things look good. I keep seeing `woocommerceanalytics` as the prefix and not `jetpack`. Thanks for checking this on your end as well! I'm marking this as resolved.

Reply:
I was able to replicate this when tracking using JS (via ajax) with a logged-in user. It was because we weren't overriding jetpack-tracking's `wp_ajax_jetpack_tracks` hook [here](https://github.com/Automattic/woocommerce-payments/pull/3796/commits/1f060f8a8f28adddd997063a95a6772532e13aa0#diff-cfa882fcd35a7427cbf7f2d3ae9f54370d02d46ca04342f3f74282df721d3b20R33). It should be fixed with https://github.com/Automattic/woocommerce-payments/pull/3796/commits/240ea404f90420402855b11c017af48d563cb641

With the above change, I also opted to separate ajax hooks specific to this implementation to avoid conflicts with other plugins that may also use `jetpack-tracking` module

Reply:
Makes sense. I missed testing with a logged-in user when trying to replicate again, thanks for pushing there! I was able to replicate without the fix, and things now work as expected with the fix 🎉 it looks good to me.